ui/packages/consul-ui/app/components/consul/intention/form/index.hbs

```diff
@@ -72,7 +72,7 @@ as |api|>
 {{/if}}
 
       <DataSource
-        @src={{concat '/' @nspace '/' @dc '/services'}}
+        @src={{concat '/*/' @dc '/services'}}
```

Comment:
why a wildcard here? 🤔

Reply:
Its to populate the service dropdowns with all services across all namespaces instead of just the services from the namespace you are in.

Ideally we'd do the 'Select a namespace, change the list of services' thing. Maybe a little improvement there we could PR at some point.